pkg/cassandra/config/v004-go-tmpl-test.cql.tmpl

```diff
@@ -0,0 +1,43 @@
+-- There are total 4 queries here
```

Comment:
I already asked - why do we need this file?

Reply:
Removed it.